internal/configs/version2/nginx-plus.transportserver.tmpl

```diff
@@ -30,10 +30,10 @@ server {
 
     proxy_pass {{ $s.ProxyPass }};
 
-    {{ if $s.HealthCheck }}
-    health_check interval={{ $s.HCInterval }} port={{ $s.HCPort }} 
-        passes={{ $s.HCPasses }} jitter={{ $s.HCJitter }} fails={{ $s.HCFails }} {{ if $s.UDP }}udp{{ end }};
-    health_check_timeout {{ $s.HCTimeout }};
+    {{ if $s.HealthCheck.Enabled }}
```

Comment:
the Enabled field is unnecessary. We can just have  this condition, which will be true if HealthCheck is not nil. 

```
{{ if $s.HealthCheck }}
   # hc stuff 
{{ end }}
```